telemetry/definitions/commonDefinitions.json

```diff
@@ -1560,6 +1560,14 @@
             ],
             "description": "Type of save executed"
         },
+        {
+            "name": "scenarioType",
+            "type": "string",
+            "allowedValues": [
```

Comment:
shall we define this as a string instead of enum? the purpose of this metric is to be low-friction so a free-form string may be better.